### ISSUE TITLE
Print thin walls not working as intended? (4.13.0+)

### ISSUE THREAD

**Issue by dtex91** [Ultimaker/Cura#11426]
### Application Version

4.13.1

### Platform

Windows 10

### Printer

Voron 2.4

### Reproduction steps

Load in a 3Dbenchy model. Whatevery options i check, the results are the bad gaps seen in the screenshot OR bad filled by cura.

### Actual results

I cant get a nice single line from cura, when there is a wall which is less than nozzle dia (0.4mm).
![1](https://user-images.githubusercontent.com/77171585/152694962-6d8f49d6-a282-4977-97c3-9b7074fec78e.png)
![2](https://user-images.githubusercontent.com/77171585/152694965-f4841ce3-61d6-4ae9-bdc4-3bbc28cf08e7.png)

### Expected results

These should be a single (thin) line, filling the gap between the walls.

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

This issue also was present version in 4.13.0

**Comment by fvrmr** [comment]
Hi @dtex91 thank you for your report.
Could you share a project file (file - save project) as a .zip here on Github?

**Comment by dtex91** [comment]
[GABS_3DBenchy.zip](https://github.com/Ultimaker/Cura/files/8015971/GABS_3DBenchy.zip)

What i have done? Start cura, put benchy in it.

**Comment by fvrmr** [comment]
Thank you for your project file. 
You need to change a bit more settings, first of all set `Fill Gaps Between Walls` to `Everywhere` and change the `Top Surface Skin Line Directions` under Experimental from [0] to [ ] (or click on the fx icon, this will restore the calculated value).

4.13.1:
![Capture(267)](https://user-images.githubusercontent.com/77000924/153004602-7bfac3a3-3c40-4a72-9ace-bf3860e60712.PNG)

Or you can use the Arachne engine: https://github.com/Ultimaker/Cura/releases/tag/Arachne_engine_beta_2 where you will have variable line width.

Arachne:
![Capture(266)](https://user-images.githubusercontent.com/77000924/153000445-d2b2abf1-1087-4b39-8ae7-45169e7db324.PNG)

Let me know if that works for you!

**Comment by dtex91** [comment]
Yep this worked. But it should be able to do both, [0] degree top surface skin lines and print thin walls, shouldnt it?
